### DISCUSSION TITLE
v5 beta 1 - Tooltip not working

### DISCUSSION THREAD

**Discussion by Croshadow123** [twbs/bootstrap#32776] [Q&A]
My bootstrap tooltip's will not work when i use "bootstrap.bundle.min.js" from v5 beta 1 (i am using  jquery-3.5.1.), but when i try to use from lets say v5 alpha 2, it works, can someone help me with that ? 

I call it "$(function () {
  $('[data-toggle="tooltip"]').tooltip();
});"

Example with the html code :
 <button class="btn btn-secondary" data-toggle="tooltip" title="This is button.">Tooltip</button>

Little more about the problem: 
When i hover with the mouse over the tooltip nothing happens, and the tooltip is not displayed. I did try all the v5 alpha versions, and on all its working as it should. Cannot find anything that could fix the problem, thats why i try here to find help.

**Comment by coliff** [comment]
You need to use `data-bs-toggle="tooltip"`
Be sure to read the migration guide when updating to a new version for changes:

> Data attributes for all JavaScript plugins are now namespaced to help distinguish Bootstrap functionality from third parties and your own code. For example, we use data-bs-toggle instead of data-toggle.

https://getbootstrap.com/docs/5.0/migration/#javascript

**Reply by smallersign** [reply]
> You need to use `bs-data-toggle="tooltip"`

It's wrong!.. **From v5 beta 1** use `-bs` after `data`

Old method `data-toggle`
New method: `data-bs-toggle`

**Reply by coliff** [reply]
I fixed my reply above. thanks :-)

**Comment by anindayuni** [comment]
> My bootstrap tooltip's will not work when i use "bootstrap.bundle.min.js" from v5 beta 1 (i am using jquery-3.5.1.), but when i try to use from lets say v5 alpha 2, it works, can someone help me with that ?
> 
> I call it "$(function () {
> $('[data-toggle="tooltip"]').tooltip();
> });"
> 
> Example with the html code :
> Tooltip
> 
> Little more about the problem:
> When i hover with the mouse over the tooltip nothing happens, and the tooltip is not displayed. I did try all the v5 alpha versions, and on all its working as it should. Cannot find anything that could fix the problem, thats why i try here to find help.

I have some problem, I changes my data-toggle to data-bs-toggle, and I used bootstrap.bundle js 5.0 beta1 but it not work

but when I using 5.0.0-alpha2/js/bootstrap.bundle
tooltips is working
